styles/style.css

```diff
@@ -6,3 +6,150 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  box-sizing: border-box;
+}
+body {
+  font-family: Circular, spotify-circular, Helvetica, Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+  width: 100vw;
+}
+
+.navbar {
+  width: 100vw;
+  height: 4.3rem;
+  position: fixed;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  padding: 0.3em 1.5em 0 1.5em;
+}
+
+ul {
+  width: 35%;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.section2 {
+  background-image: url("../images/landing.jpg");
+  background-size: cover;
```

Comment:
You can also put a   background-position: center;
This will center your image and make it look a bit nicer:)